docker/Dockerfile.deepstream

```diff
@@ -45,7 +45,7 @@ RUN python setup.py bdist_wheel && rm -rf _skbuild
 FROM base_builder AS savantboost_build
```

Comment:
There are 2 TODOs above. 

The first one (`TODO: Update library code, CMakeLists (see build process warnings)`) will not be relevant after your refactoring. 

The second `TODO: Check Jetson requirements (Xavier t194, Orin t234)` is still actual. What do you think about installing nvidia-l4t-multimedia from the t194 repo, would there be any problems using savantboost on Orin (t234)? What components do we really need to build savantboost on Jetson?

Reply:
Without the nvidia-l4t-multimedia library, you can't build a library. On Orin I checked it builds fine, age-gender example using custom preprocessing works fine.

Reply:
If there is no problem with Orin, remove the second TODO too.